src/libraries/Microsoft.Extensions.Configuration.Ini/src/IniStreamConfigurationProvider.cs

```diff
@@ -23,16 +23,16 @@ public class IniStreamConfigurationProvider : StreamConfigurationProvider
         /// </summary>
         /// <param name="stream">The stream of INI data.</param>
         /// <returns>The <see cref="IDictionary{String, String}"/> which was read from the stream.</returns>
-        public static IDictionary<string, string> Read(Stream stream)
+        public static IDictionary<string, string?> Read(Stream stream)
         {
-            var data = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+            var data = new Dictionary<string, string?>(StringComparer.OrdinalIgnoreCase);
```

Comment:
When will this dictionary contain `null` values? I don't see it how `null` gets in here.

Reply:
Oh shoot, thanks! This was left from times when I thought, `ReadLine` can return `null`

Reply:
Results from `Read` are assigned to `Data` (that has `IDictionary<string, string?>` type), so `Read` also needs to return `IDictionary<string, string?>`

Reply:
Ok, that makes sense. This is `public` API, but its main use case is to create the `Data` dictionary in configuration, which is `string, string?`, so it makes sense for this public API to be consistent.